Toolbar customization/InsertToolbarItems/FileService.cs

```diff
@@ -0,0 +1,114 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Xml.Linq;
+
+namespace InsertToolbarItems
+{
+    /// <summary>
+    /// Provides the functionality to open and save PDF files. 
+    /// </summary>
+    public partial class FileService
+    {
+        /// <summary>
+        /// Shows a file picker and lets the user choose the required PDF file. 
+        /// </summary>
+        /// <returns>The data of the PDF file that is chosen by the user.</returns>
+        public async static Task<PdfFileData?> OpenFileRequest(string fileExtension)
+        {
+            // Define platform-specific file types for the file picker.
+            FilePickerFileType pdfFileType = new FilePickerFileType(new Dictionary<DevicePlatform, IEnumerable<string>>{
+                        { DevicePlatform.iOS, new[] { $"com.adobe.{fileExtension}" } },
+                        { DevicePlatform.Android, new[] { $"application/{fileExtension}" } },
+                        { DevicePlatform.WinUI, new[] { fileExtension } },
+                        { DevicePlatform.MacCatalyst, new[] { fileExtension } },
+                    });
+
+            // Configure the file picker options.
+            PickOptions options = new()
+            {
+                PickerTitle = "Please select a PDF file",
+            };
+
+            // Set file type for the pick option to pick the file.
+#if !ANDROID
+            options.FileTypes = pdfFileType;
+#else
+            if (fileExtension == "pdf")
+                options.FileTypes = pdfFileType;
+#endif
+
+            // Call the method to pick the file and return the result.
```

Comment:
Use proper null condition instead of null escape operator.
<img width="375" height="131" alt="image" src="https://github.com/user-attachments/assets/2d6aaa27-d308-4962-b509-8599b68dd83f" />